yoti-sdk-spring-boot-example/src/main/java/com/yoti/api/examples/springboot/YotiLoginController.java

```diff
@@ -1,30 +1,46 @@
 package com.yoti.api.examples.springboot;
 
+import com.yoti.api.client.ActivityDetails;
```

Comment:
I think IDEA auto-reordered this, is this right?

Reply:
not ideal, but alright.